config/locales/forms/change_email.yml

```diff
@@ -15,3 +15,4 @@ en:
             submission_email:
               blank: Enter an email address
               invalid_email: Enter an email address in the correct format, like name@example.com
+              non_govuk_email: Enter an email address ending in .gov.uk
```

Comment:
Didn't we say we were going to validate it on `@gov.uk` rather than `.gov.uk`? I may be misremembering

Reply:
Think it hads to be `.gov.uk` as `@gov.uk` would exclude Insolvency and our emails